README.md

```diff
@@ -1,2 +1,85 @@
 # PharoDEVS
-An implementation of discrete event system specification
+An Pharo based implementation of [Discrete Event System Specification](https://en.wikipedia.org/wiki/DEVS) - formalism for modelling and simulation by B. Ziegler.
+
+## Loading project
+Install [Pharo latest stable version](https://pharo.org/download) e.g. from command line:  
+`wget -O- https://get.pharo.org/64 | bash` 
+
+Run in Pharo Playground:
+```
+Metacello new
+ baseline: 'PharoDEVS';
+ repository: 'github://ApptiveGrid/PharoDEVS:main/src';
+ load
+```
+
+## Running examples
+TODO
+
+## History
+Current implementation is Pharo port and continuation of original [SmallDEVS library](http://perchta.fit.vutbr.cz:8000/projekty/10) implemented in Squeak 4.3 at [Department of Intelligent Systems on FIT BUT](https://www.fit.vut.cz/units/uits/staff/.en) (namely doc. Ing. Vladimír Janoušek Ph.D., Ing. Elöd Kironský).
```

Comment:
This is not really true because there is also code from my own start of DES system. So I will adjust the text later, I will merge as it is only details and I don't care that much. As long as the work does not go into the readme but also in code ;)